lib/canvas_api/rb_graphql_helpers.rb

```diff
@@ -161,11 +174,11 @@ def is_basic_type(type)
     end
 
     def no_brackets(str)
-      str.gsub("[", "").gsub("]", "")
+      str.gsub("[", "").gsub("]", "").gsub(".", "")
```

Comment:
This could be rewritten as:
```suggestion
      str.tr('[].', '')
```
I think that might improve readability.